apps/web/.env.example

```diff
@@ -50,4 +50,7 @@ AWS_SECRET_NAME=
 AWS_REGION=
 
 # ODP
-ODP_ROLE_GROUP_ID=
\ No newline at end of file
+ODP_ROLE_GROUP_ID=
+
+# List of comma seperated emails that will see the new edit study functionality 
+EDIT_STUDY_ALLOW_LIST=
```

Comment:
Do we definitely want to make this an environment variable requiring redeployment?

It seems like this is something we would change frequently during UAT/Operations.
Should this be driven from database state instead?

Reply:
This was my steer - it follows the pattern of the NOTIFY_ALLOW_LIST.
The idea is this is only used for MVP - i.e. a handful of testers/devs, to test integrations in prod ahead of go live, then removed completely when we launch - db feels a bit overkill for throwaway code? especially if we have rolling deploys? But happy to take a steer if you feel strongly or I've missed something!

Reply:
Sure. If it is temporary that makes sense - though it still seems clunky if we need to update the list in response to a client request.

If we are already considering using IDG groups for the ODP dashboard access could that be an option here too?

Reply:
Thank you. This has now been updated. I've followed the approach we do for the ODP role. 

I've added a const with the given role `'SEPhase2_Beta_Tester'` and we simply check if the user has this.

Reply:
The final push to add the role to the user session has been added. 

I've tested it locally and it works.